webtool/static/js/fourcat.js

```diff
@@ -1670,11 +1689,11 @@ const ui_helpers = {
      * @param force_close  Assume the event is un-toggling something regardless of current state
      */
     toggleButton: function (e, force_close = false) {
-        if (!e.target.hasAttribute('type') || e.target.getAttribute('type') !== 'checkbox') {
+        if ((!e.target.hasAttribute('type') || e.target.getAttribute('type') !== 'checkbox') && typeof e.preventDefault === "function") {
             e.preventDefault();
         }
 
-        let target = '#' + $(this).attr('aria-controls');
+        let target = '#' + $(e.target).attr('aria-controls');
```

Comment:
I am not sure the intent of this change, but it appears to cause an issue with inner elements not triggering events. For example, the "More" button to expand run processors does not work if you click the icon or the word "More", just the edges of the button!

![image](https://github.com/digitalmethodsinitiative/4cat/assets/32108944/6bc6f354-d025-40dd-8e71-e0973c6707d2)